.stylelintrc

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
I think it's good to have the patches about installing `stylelint` packages in this commit.
If it's too hard to separate them, I think it's good to notice this kind of things in the future :P